components/buttons/buttons.css

```diff
@@ -0,0 +1,179 @@
+/* ---default button */
+.btn {
+    border: none;
```

Comment:
border: none can be added in the reset css.

Reply:
Its creating some problem with other buttons that's why kept it this way. I can have it in reset too thanks for suggestion